docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -102,26 +109,26 @@ init`.
 
 There are pod network implementations where the master also plays a role in
 allocating a set of network address space for each node.  When using
-[flannel](https://github.com/coreos/flannel) as the [pod network](#pod-network)
+[Flannel](https://github.com/coreos/flannel) as the [pod network](#pod-network)
 (described in step 3), specify `--pod-network-cidr=10.244.0.0/16`. _This is not
```

Comment:
No, that's the Service CIDR (default 10.96.0.0/12), not the Pod CIDR (depends on net provider)